src/vs/workbench/browser/parts/compositePart.ts

```diff
@@ -388,6 +392,18 @@ export abstract class CompositePart<T extends Composite> extends Part {
 		const titleArea = append(parent, $('.composite'));
 		titleArea.classList.add('title');
 
+		// MEMBRANE: back to Navigator button
+		const backToNavigator = document.createElement('a');
+		backToNavigator.classList.add('back-to-membrane-navigator', 'codicon', 'codicon-x', 'action-item', 'action-label');
```

Comment:
There are other [codeicon options](https://code.visualstudio.com/api/references/icons-in-labels#icon-listing) we could use here. E.g. `reply`, `globe`, `triangle-left`, `home`